<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  
    <title>Our funded research projects - Biodata Analysis Group</title>
  

  <meta name="description" content="The Biodata Analysis Group website">
  <meta name="author" content="Fotis Psomopoulos">

  <!-- CSS -->
  <!-- Bootstrap is included in main.css -->
  <link rel="stylesheet" href="/assets/theme/css/main.css">
  <link rel="stylesheet" href="/assets/theme/css/font-awesome.min.css">
  <link rel="stylesheet" href="/assets/theme/css/pygments.css">

  <!-- Fonts -->
  <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i&display=swap" rel="stylesheet">

  <!-- Feeds -->
  <link rel="alternate" type="application/rss+xml" title="Biodata Analysis Group - RSS feed" href="https://biodataanalysisgroup.github.io/feed.xml" >
  <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-17296390-13"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-17296390-13');
</script>

</head>


  <body>
    <nav class="navbar navbar-expand-lg navbar-light bg-light" id="main-nav">
  <div class="container">
    <a class="navbar-brand d-flex align-items-center" href="/">
      
        <img src="/assets/img/groupLogo/helix.png" class="mr-2">
      
      Biodata Analysis Group
    </a>
    
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-menu">
        Menu <i class="fa fa-bars"></i>
      </button>
      <div class="collapse navbar-collapse" id="navbar-menu">
        <ul class="navbar-nav ml-auto">
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/" >Home</a>
            </li>
          
        
          
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown">
                The Group
              </a>
              <div class="dropdown-menu">
              
                
                  <a class="dropdown-item" href="/team/" >Team</a>
                
              
              </div>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/publications/" >Publications</a>
            </li>
          
        
          
            <li class="nav-item dropdown">
              <a class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown">
                Research
              </a>
              <div class="dropdown-menu">
              
                
                  <a class="dropdown-item" href="/activities/" >Activities</a>
                
              
                
                  <a class="dropdown-item active" href="/projects/" >Funded Projects</a>
                
              
                
                  <a class="dropdown-item" href="/collaborations/" >Collaborations</a>
                
              
                
                  <div class="dropdown-divider"></div>
                
              
                
                  <h6 class="dropdown-header">External resources</h6>
                
              
                
                  <a class="dropdown-item" href="https://github.com/BiodataAnalysisGroup" target="_blank">Lab GitHub</a>
                
              
              </div>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/software/" >Software</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/blog/" >News</a>
            </li>
          
        
          
            <li class="navbar-item">
              <a class="nav-link" href="/about/" >About</a>
            </li>
          
        
        </ul>
      </div>
    
  </div>
</nav>


    <main>
      
  <header class="jumbotron bg-image" style="background-image: url('https://images.unsplash.com/photo-1572177812156-58036aae439c?auto=format&w=2000')">

    <div class="container">
      <div class="row">
        <div class="col-md-10 col-lg-8 mx-auto">
          <h1>Our funded research projects</h1>
          
          <div class="lead"><p>Listing our funded projects here</p>
</div>
          
          

          <div class="categories">
            
          </div>
        </div>
      </div>
    </div>
  </header>


      <div class="container content">

        <div class="row">
  <div class="col-md-10 col-lg-8 mx-auto">

    <p><a href="#ongoing"><strong>Ongoing</strong></a>   <a href="#completed"><strong>Completed</strong></a>  </p>

<p><br /></p>

<h1 id="ongoing">Ongoing</h1>
<hr style="border: 1px solid #cfcfcf;" />

<h2 id="raise-connect">RAISE-Connect</h2>

<p><img src="/assets/img/collaborations/raise_connect_logo.png" alt="RAISE-Connect logo" style="width:300px;" /></p>

<p><em>Enabling and Connecting Interoperable FAIR Digital Objects</em></p>

<ul>
  <li><strong>Duration</strong>: 2026-2029</li>
  <li><strong>Funder</strong>: HORIZON-INFRA-2025-01-EOSC-02</li>
  <li><strong>Description</strong>: Despite significant progress in open data, code sharing, and implementation of FAIR principles, there is no automated, universal
mechanism capable to “FAIR-ify” DOs across large cross-domain repositories and data spaces. RAISE-CONNECT addresses this
challenge by providing AI-powered, domain-agnostic connectors that can “FAIR-ify” DOs (i.e., data, code, experiments, experimental
results and FDOs) and make them interoperable both within and across repositories. Leveraging existing EOSC-RAISE tools, it offers
researchers and other stakeholders an entry point to access, share, and process FDOs with no effort. RAISE-CONNECT establishes an
architectural framework for seamless creation, transformation, and integration of FDOs. At its core, it introduces an automated “FAIRification”
service to continuously enhance DOs by searching across repositories for missing FAIR elements. This ensures that every DO
can be transformed into an FDO that is discoverable, interoperable, and reusable. To maximise usability and uptake, RAISE-CONNECT
lets users interact with DOs through natural language prompts, leveraging LLMs: users can “ask” for datasets, scripts, compare results,
all without technical overhead. A model of Common Repository Connector acts as ”template” abstraction layer to bridge
heterogeneous repositories, while different specific implementations of the it will be developed for well-known Data Spaces and DO
repositories.</li>
</ul>

<p><br /></p>

<h2 id="renai">RenAI</h2>

<!-- <img src="/assets/img/collaborations/RenAI-logo.png" alt="RenAI logo" style="width:200px;"/>
-->

<p><em>Responsible AI Infrastructures for scientific excellence</em></p>

<ul>
  <li><strong>Duration</strong>: 2026-2029</li>
  <li><strong>Funder</strong>: HORIZON-INFRA-2025-01-EOSC-05</li>
  <li><strong>Description</strong>: RenAI is set out to deliver an end-to-end GenAI ecosystem in EOSC that embeds domain-specific, pre-trained models to support advanced tools for data curation and synthetic data generation, and the RenAI coResearcher, an intuitive, agentic AI companion that empowers researchers across disciplines and skill levels to seamlessly access and apply trustworthy, compliant, and high-impact GenAI workflows from RenAI, accelerating scientific discovery within EOSC and beyond. To break down barriers and address the fragmented, unequal access to GenAI in research, RenAI builds on five interconnected pillars: (i) Scientific Pretrained Models; (ii) AI-Readiness and Transformation Infrastructure; (iii) RenAI coResearcher; (iv) Legal and Ethical Compliance Hub; (v) Community Upskilling</li>
</ul>

<p><br /></p>

<h2 id="eosc-arena">EOSC-ARENA</h2>

<p><img src="/assets/img/collaborations/arena_logo.png" alt="ARENA logo" style="width:300px;" /></p>

<p><em>AI Research Enhancement through Networked Agents</em></p>

<ul>
  <li><strong>Duration</strong>: 2026-2029</li>
  <li><strong>Funder</strong>: HORIZON-INFRA-2025-01-EOSC-05</li>
  <li><strong>Description</strong>: EOSC-ARENA goes beyond the simple application of GenAI solutions to specific cases, focusing on delivering an ambitious agentic AI system, enhancing researchers with tools that are not only advanced in terms of GenAI but also adaptive and responsive to the dynamic needs of scientific inquiry (via agents). Our vision extends beyond traditional GenAI applications by working on three different foundation areas: (i) A sophisticated GenAI agentic system, with a strong focus on scientific robustness; (ii) a community building and sound training program focused on responsible usage, enabling researchers to perform informed decisions, aligned with EU values from an ethical and human-centric perspective; and (iii) a comprehensive set of GenAI infrastructure services. All together, these three areas serve as enablers in order to deliver the final sovereign AI ecosystem to enable open science and collaborative Human-AI work.</li>
</ul>

<p><br /></p>

<h2 id="b1mgplus">B1MGplus</h2>

<p><img src="/assets/img/collaborations/B1MG-logo.png" alt="B1MG logo" style="width:200px;" /></p>

<p><a href="https://b1mgplus.onemilliongenomes.eu/"><em>Beyond 1 Million Genomes plus: Driving the future of evidence-based healthcare</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2025-2028</li>
  <li><strong>Funder</strong>: DIGITAL-2024-CLOUD-DATA-AI-06-GENOME</li>
  <li><strong>Description</strong>: The Beyond 1 Million Genomes plus (B1MGplus) project is helping create a European cross-border network of genomic and corresponding clinical data to improve healthcare outcomes. The project provides coordination and support to the 1+ Million Genomes initiative (1+MG), which is the commitment of 25 EU countries plus Norway to enable secure access to genomic and corresponding clinical data across Europe, supporting research, health policy and personalised healthcare.</li>
</ul>

<p><br /></p>

<h2 id="candle">CANDLE</h2>

<p><img src="/assets/img/collaborations/candle_logo.png" alt="CANDLE logo" style="width:200px;" /></p>

<p><a href="https://candle-project.eu/"><em>National CAncer data Node DeveLopErs</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2025-2028</li>
  <li><strong>Funder</strong>: HORIZON-MISS-2024-CANCER-01</li>
  <li><strong>Description</strong>: Coordinated by Health-RI and with a consortium of 40 partners from 20 European countries, the project will support Member States and Associated Countries in setting up National Cancer Data Nodes. These Nodes will promote the development of standardized services for managing and sharing cancer data nationally and across borders, connecting different national actors.
Despite vast amounts of cancer-related data in Europe, fragmentation and lack of interoperability hinder progress in cancer research and treatment. CANDLE addresses this by establishing trusted nodes to connect data, tools, and expertise across borders. These National Cancer Data Nodes will facilitate access to cancer data from across Europe. Cancer researchers can then access more data than is currently accessible – and help Europe better understand cancer. This can fuel the development of new cancer treatments, diagnostics, and better information to patients, survivors, and carers – and much more.
CANDLE will work toward this by:
    <ul>
      <li>Promoting the development and implementation of National Cancer Data Nodes in participating countries, to support users in handling their data with the right applications and underlying infrastructure.</li>
      <li>Promoting harmonisation and standardisation of cancer data.</li>
      <li>Supporting data discoverability, accessibility, and reuse in line with FAIR principles.</li>
      <li>Enabling cross-border, GDPR-compliant data sharing for research and innovation.</li>
      <li>Benefiting from the European Health Data Space and contributing to the UNCAN.eu European Cancer Data Infrastructure.</li>
    </ul>
  </li>
</ul>

<p><br /></p>

<h2 id="late-aya">LATE-AYA</h2>

<p><img src="/assets/img/collaborations/late_aya_logo.png" alt="LATE-AYA logo" style="width:200px;" /></p>

<p><a href="https://www.late-aya.eu/"><em>Understanding and addressing LATE-effects of treatment of AYA cancer survivors with AI-based digital phenotyping and non-invasive holistic approach</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2025-2030</li>
  <li><strong>Funder</strong>: HORIZON-MISS-2024-CANCER-01-05</li>
  <li><strong>Description</strong>: Adolescents and Young Adults (AYAs) face a distinct set of challenges, including increased risks of secondary cancers, cardiovascular diseases, infertility, and mental health issueslike anxiety and depression. Oncology programs delivering care to AYAs provide survivors with individualised survivorship care plans at the end of treatment, to accompany follow-up with their primary care providers. However, adherence to these plans is often scarce, due to differences in AYA survivors’ age, sex, cancer treatment received, and socioeconomical conditions. Knowledge and understanding of such factors and of related late effects and impacts on quality of life is fundamental to providing effective survivorship care. The ubiquitous use of mobile devices by AYAs offers an opportunity to increase knowledge and understanding of AYAs cancer survivors’ health status and behaviours, allowing the modelling of late effects and quality of life trajectories in the different age subgroups. Digital health interventions overcome many barriers to AYA participation in survivorship programs. LATE-AYA seeks to fill this gap by empowering AYAs to better manage their health and well-being post-cancer, by developing an AI-driven digital phenotyping platform for managing late effects (LE) of cancer treatment. The project will implement a holistic, non-invasive approach through digital tools such as smartphones and wearables to monitor physical, psychological, and social well-being. The project will focus on preventive health behaviours, psychological support, and social reintegration, providing personalized care through digital interventions. LATE-AYA will contribute to long-term quality of life improvements, facilitate early detection of LE, and provide data-driven insights on the impact of lifestyle choices on health outcomes. This project is supported by a diverse consortium of European institutions and will leverage the UNCAN.eu platform to share data and models, fostering wider collaboration across the EU. This action is part of the Cancer Mission cluster of projects on “Quality of life (AYA)”.</li>
</ul>

<p><br /></p>

<h2 id="metaplantcode">METAPLANTCODE</h2>

<p><img src="/assets/img/collaborations/metaplantlogo.png" alt="METAPLANTCODE logo" style="width:200px;" /></p>

<p><a href="http://www.metaplantcode.eu/"><em>Harmonizing plant metabarcoding pipelines in Europe to support monitoring activities in the field of plants and their functional organismic networks</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2024-2027</li>
  <li><strong>Funder</strong>: 2022–2023 BiodivMon</li>
  <li><strong>Description</strong>: METAPLANTCODE presents a unique collaborative and transnational approach to test, optimise, harmonise and recommend best practices for plant metabarcoding for samples with varying degrees of species complexity, contamination, and DNA degradation using case studies across Europe. The innovative combination of accelerated molecular plant monitoring and the integration of diverse biodiversity data with optimised and automated pipelines will fill knowledge gaps on the state of biodiversity, interdependencies, and dynamics. The METAPLANTCODE project aims to test and optimise pan-European case studies on metabarcoding, provide best practice recommendations, optimise analysis pipelines for species identification, and create easy-to-use reference databases. The project will identify and specify gaps, publish best practice documents on FAIR data publishing of plant metabarcode data to GBIF and the INSDC databases, and implement ELIXIR-compatible multimodal DL models in novel tools for stand-alone metabarcoding analyses using different data sources. The project will also enhance species identification accuracy through GBIF records and metadata, and map regional, national, and international botanical taxonomic checklists, red lists, and floras to the Catalogue of Life (COL) through COL ChecklistBank.</li>
</ul>

<p><br /></p>

<h2 id="synthia">SYNTHIA</h2>

<p><img src="/assets/img/collaborations/synthia.jpg" alt="SYNTHIA" style="width:150px;" /></p>

<p><a href="http://www.ihi-synthia.eu/"><em>Synthetic Data Generation framework for integrated validation of use cases and AI healthcare applications</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2024-2029</li>
  <li><strong>Funder</strong>: HORIZON-JU-IHI-2023-05-04</li>
  <li><strong>Description</strong>: SYNTHIA is an ambitious collaboration between public and private institutions to facilitate the responsible use of Synthetic Data (SD) in healthcare applications. The project will improve the methodological and technical aspects of SD Generation (SDG) by developing new techniques and advancing established ones for different data modalities, including genomics and imaging, to improve the generation of realistic multimodal and longitudinal data. This project will provide the research community with approaches for transparent benchmarking of alternative SDG methods for specific applications, identify and establish evaluation metrics and methodologies, and contribute to the standardisation of an evaluation assessment framework for SD. Robust evidence of SD applicability in a set of use cases across a broad spectrum of medical conditions will be crucial to demonstrate the potential of SD to accelerate data-driven solutions of equivalent quality to those derived from real patient data. Furthermore, legal and regulatory implications of SD use will be analysed with the aim of delivering an assurance framework to guide secure SD utilization in healthcare. These significant breakthroughs will be implemented through the open SYNTHIA federated platform, facilitating responsible SD use by the health research community. The platform will facilitate users´ long-term access to extensively validated, reusable synthetic datasets, as well as to SDG workflows and SD assessment frameworks. The federated infrastructure will rely on extended open-source frameworks for interoperability with other data-sharing infrastructures in the context of the European Health Data Space. A multidisciplinary collaboration of SDG developers, FAIR data experts, clinical researchers, developers of therapies and data-based tools, legal experts, socio-economic analysts, regulatory, policy advocacy, and communication experts will provide a 360º vision on how to advance healthcare applications through SD use.</li>
</ul>

<p><br /></p>

<h2 id="ahead">AHEAD</h2>

<p><img src="/assets/img/collaborations/AHEADLogo.png" alt="AHEAD logo" style="width:150px;" /></p>

<p><a href="https://ahead-msca.eu/"><em>AI for Health: Evaluation of Applications &amp; Datasets</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2025-2028</li>
  <li><strong>Funder</strong>: HORIZON-MSCA-2023-SE-01</li>
  <li><strong>Description</strong>: The proliferation of Artificial Intelligence (AI) is on the rise, and yet, there remains a lack of consensus on standardised approaches for responsible implementation. In Europe, there is a strong push to regulate AI, particularly in high-impact societal applications such as biomedicine and healthcare. The objective of AHEAD is to create a transdisciplinary, diverse and global community drawing on the expertise of professionals in biomedicine, ethics, AI development, software engineering, sociology, psychology, law, gender studies, and relevant stakeholders, dedicated to the ongoing effort of tackling challenges and setting important standards in an ever-changing landscape. Together, the community will gather, consolidate, and put into practice established principles and guidelines, building a methodology and a platform to evaluate AI-based systems for biomedical and healthcare applications to ensure the compliance with legal, ethical, technical, scientific, and regulatory standards. The project will extend the fucntionalities of OpenEBench, ELIXIR’s benchmarking platform for bioinformatics tools, to provide a platform for these assessments. The community will also establish an observatory to collect and curate existing guidelines, best practices, and principles for AI assessment in biomedicine and healthcare across multiple disciplines (technical, societal, legal, etc.). AHEAD will  encourage responsible AI innovation through constant collaboration and knowledge exchanges, to continuously meet the demands of the fast-changing AI landscape and ensure that quality healthcare for all people is never compromised and only enhanced. The proposed project is the beginning of a sustained endeavour that would greatly benefit from mobility grants to enhance community bonds and foster collaborative efforts toward this goal.</li>
</ul>

<p><br /></p>

<h2 id="odyssey">ODYSSEY</h2>

<p><img src="/assets/img/collaborations/Odyssey_logo.png" alt="ODYSSEY" style="width:200px;" /></p>

<p><a href="https://elixir-europe.org/how-we-work/scientific-programme/commissioned-services/science/bfsp/odyssey"><em>Connecting molecular and geographical biodiversity data</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2025-2026</li>
  <li><strong>Funder</strong>: ELIXIR Scientific Programme 2024-2028 (BFSP)</li>
  <li><strong>Description</strong>: Odyssey is a web portal in the form of a user-friendly interface that will allow researchers, educators and citizens to navigate into the world of molecular biodiversity using Greece and Norway as case studies, two countries with characteristic and unique biodiversity wealth, representative for Mediterranean and Nordic types of ecosystems respectively. Based on existing sources of information and prototype applications that are available for specific regions and taxa, this project aims to link actual efforts and develop a new interface to offer diverse functionalities for data exploration and analysis such as descriptive statistics, graphs, maps, customizable data filters, and dynamic visualisations.</li>
</ul>

<p><br /></p>

<h2 id="everse">EVERSE</h2>

<p><img src="/assets/img/collaborations/EOSC_Everse.jpg" alt="EVERSE" style="width:230px;" /></p>

<p><a href="https://everse.software/"><em>European Virtual Institute for Research Software Excellence</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2024-2027</li>
  <li><strong>Funder</strong>: HORIZON-INFRA-2023-EOSC-01-02</li>
  <li><strong>Description</strong>: The EVERSE project aims to create a framework for research software and code excellence, collaboratively designed and championed by the research communities across five EOSC Science Clusters and national Research Software Expertise Centres, in pursuit of building a European network of Research Software Quality and setting the foundations of a future Virtual Institute for Research Software Excellence. This framework for research software excellence will incorporate aspects involving community curation, quality assessment, and best practices for research software. This collective knowledge will be captured in the Research Software Quality toolkit (RSQkit), a knowledge base to gather and curate expertise that will contribute to high-quality software and code across different disciplines.</li>
</ul>

<p><br /></p>

<h2 id="scilake">SciLake</h2>

<p><img src="/assets/img/collaborations/scilake.png" alt="SciLake" style="width:120px;" /></p>

<p><a href="https://scilake.eu/"><em>Democratising and making sense out of heterogeneous scholarly content</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2023 – 2026</li>
  <li><strong>Funder</strong>:  HORIZON-INFRA-2021-EOSC-01-04</li>
  <li><strong>Description</strong>: SciLake’s mission is to build upon the OpenAIRE ecosystem and EOSC services to (a) facilitate and empower the creation, interlinking and maintenance of SKGs and the execution of data science and graph mining queries on top of them, (b) contribute to the democratization of scholarly content and the related added value services implementing a community-driven management approach, and (c) offer advanced, AI-assisted services that exploit customised perspectives of scientific merit to assist the navigation of the vast scientific knowledge space. In brief, SciLake will develop, support, and offer customisable services to the research community following a two-tier service architecture. First, it will offer a comprehensive, open, transparent, and customisable scientific data-lake-as-a-service (service tier 1), empowering and facilitating the creation, interlinking, and maintenance of SKGs both across and within different scientific disciplines. On top of that, it will build and offer a tier of customisable, AI-assisted services that facilitate the navigation of scholarly content following a scientific merit-driven approach (tier 2), focusing on two merit aspects which are crucial for the research community at large: impact and reproducibility. The services in both tiers will leverage advanced AI techniques (text and graph mining) that are going to exploit and extend existing technologies provided by SciLake’s technology partners. Finally, to showcase the value of the provided services and their capability to address current and anticipated needs of different research communities, four scientific domains (neuroscience, cancer research, transportation, and energy) have been selected to serve as pilots. For each, the developed services will be customised, to accommodate differences in research procedures, practices, impact measures and types of research objects, and will be validated and evaluated through real-world use cases.</li>
</ul>

<p><br /></p>

<h2 id="ml4ngp">ML4NGP</h2>

<p><img src="/assets/img/collaborations/ML4NGP.png" alt="ML4NGP" style="width:150px;" /></p>

<p><a href="https://www.cost.eu/cost-action/non-globular-proteins-in-the-era-of-machine-learning/"><em>CA21160 - Non-globular proteins in the era of Machine Learning</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2022 - 2026</li>
  <li><strong>Funder</strong>: <a href="https://www.cost.eu/">Cost - European Cooperation in Science &amp; Technology</a></li>
  <li><strong>Description</strong>: Protein structure prediction has long been considered the “Holy Grail” of structural biology. The recent success of AlphaFold has ushered in a new era of highly accurate structure prediction, bringing to light the secrets hidden in the three-dimensional structures of globular proteins, increasing our understanding about their structural features and molecular function. However, a large proportion of the proteomes from all domains of life are rich in sequences that do not fold into regular structures, commonly known as non-globular proteins (NGPs). NGPs comprise intrinsically disordered regions, repeats, low-complexity sequences, aggregation-prone and phase-separating sequences, and are implicated in a range of age-related diseases. Their heterogeneous structural states and low sequence complexity challenge current experimental structure determination techniques and machine learning (ML) methods for structure prediction, making the molecular understanding of their sequence-structure-dynamics-function relationship difficult. The recent improvements of ML approaches and advances in determining NGP structural ensembles call for a timely re-assessment of the interplay between experiments and computation. The ML4NGP Action aims to establish an interdisciplinary pan-European network to favour this interplay, fostering experimental frameworks designed to provide information to computational methods, and novel computational methods developed, trained and benchmarked with experimental data. ML4NGP will enhance the primary experimental data generation (WG1), promote integrative structural biology approaches (WG2), benchmark the state-of-the-art ML methods (WG3) and improve the functional characterization of NGPs (WG4). The Action will support its scientific objectives through policies that sustain free knowledge exchange, inclusiveness and training of young researchers who will lead future innovations in this field.</li>
</ul>

<p><br /></p>

<h1 id="completed">Completed</h1>
<hr style="border: 1px solid #ccc;" />

<h2 id="eosc4cancer">EOSC4Cancer</h2>

<p><img src="/assets/img/collaborations/EOSC4Cancer.png" alt="EOSC4Cancer" style="width:200px;" /></p>

<p><a href="https://eosc4cancer.eu/"><em>A European-wide foundation to accelerate Data-driven Cancer Research</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2023-2026</li>
  <li><strong>Funder</strong>: HORIZON-INFRA-2021-EOSC-01-06</li>
  <li><strong>Description</strong>: Cancer complex nature requires integration of advanced research data across national boundaries to enable progress. Indeed, the Horizon Europe mission board for cancer has identified access to data, knowledge and digital services - accessible across the European Research Area through federated infrastructures - as a key enabling condition for success. The better we organise cancer data across Europe, the better and faster we can bring the fruits of new biological and technical innovations to the benefit of EU citizens/patients. EOSC4Cancer will make cancer genomics, imaging, medical, clinical, environmental and socio-economics data accessible, using and enhancing existing federated and interoperable systems for securely identifying, sharing, processing and reusing FAIR cancer data across borders, and it will offer them via community-driven analysis environments. EOSC4Cancer provision of well curated datasets will be essential for advanced analytics and computational methods to be reproducible and robust, including machine learning and artificial intelligence approaches. EOSC4Cancer use-cases will cover the patient journey from cancer prevention to diagnosis to treatment, laying the foundation of data trajectories and workflows for future cancer mission projects. EOSC4Cancer brings together a comprehensive consortium of cancer research centres, research infrastructures, leading research groups, hospitals and supercomputing centres from 14 European countries. To make the developments sustainable, these will be offered as part of the research infrastructures partners services portfolio, in connection with the EOSC ecosystem and to serve the European Cancer Mission, which will be possible via the engagement with large international coalitions, e.g. ICGC-Argo, GA4GH, 1+MG/B1MG, Cancer Core Europe, European Cancer Information System, European Network of Cancer Registries, Innovative Partnership for Action Against Cancer Joint Action and patients/survivors associations.</li>
</ul>

<p><br /></p>

<h2 id="genoptics">GenOptics</h2>

<p><img src="/assets/img/collaborations/GenOptics_logo.png" alt="GenOptics" style="width:150px;" /></p>

<p><a href="https://genoptics.github.io/"><em>Large Scale bio-data Visual analytics platform</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2020 - 2023</li>
  <li><strong>Funder</strong>: <a href="http://www.gsrt.gr/">General Secretariat for Research and Technology (GSRT)</a></li>
  <li><strong>Description</strong>: GenOptics is a 30-month national project (2020-2023) that will implement a new platform for integrating, analyzing and visualizing multi-omics and other clinico-biological data, by extending the functionality of established bioinformatic analysis tools through an interactive visual dashboard. Visual analytics, namely the simultaneous use of computational data analysis tools with interactive visualizations, is a powerful way of combining algorithms with an intuitive, user-friendly interactive environment in order to extract knowledge from large scale bio-data. Although visual analytics could contribute to significantly reducing the gap between available data and discovery of new knowledge through targeted studies, there is no widely used platform that provides this functionality to the scientific community. GenOptics will allow visual analytics of multi-omics data through a modular system that will integrate multiple interoperable analysis tools into a software platform, leveraging established international technologies and software suites such as Docker and R/Bioconductor.</li>
</ul>

<p><br /></p>

<h2 id="gallantries">Gallantries</h2>

<p><img src="/assets/img/projects/gallantries-logo-with-word-inverted-large.png" alt="Gallantries" style="width:150px;" /></p>

<p><a href="https://gallantries.github.io/"><em>When Galaxy meets Carpentries to develop and deliver open and scalable training in life sciences</em></a></p>

<ul>
  <li><strong>Duration</strong>: 2020 - 2023</li>
  <li><strong>Funder</strong>: <a href="https://ec.europa.eu/programmes/erasmus-plus/node_en">Erasmus+</a></li>
  <li><strong>Description</strong>: The <a href="https://gallantries.github.io/">Gallantries project</a> has been funded by <a href="https://ec.europa.eu/programmes/erasmus-plus/node_en">Erasmus+</a> to develop modular, broadcast-ready bioinformatics lessons over the next 3 years, aiming in particular for a hybrid form of training delivery. Partners and associated members of the project include several ELIXIR Nodes and Communities, and are dedicated to developing high quality bioinformatics training by combining the experience of Carpentries trainers with the Galaxy platform. Saskia Hiltemann &amp; Helena Rasche (<a href="https://erasmusmc.nl/">ErasmusMC</a>, ELIXIR-NL), Fotis Psomopoulos (<a href="https://inab.certh.gr/">INAB, CERTH</a>, ELIXIR-GR), Bérénice Batut (<a href="https://uni-freiburg.de/">University of Freiburg</a>, ELIXIR-DE), Anthony Bretaudeau (<a href="https://www.inrae.fr/en">INRAE</a>, ELIXIR-FR), and Yvan Le  Bras (<a href="https://www.mnhn.fr/en">MNHN</a>, ELIXIR-FR) and their teams will be working together to develop, improve, and deliver quality, reusable, modular, and pandemic-friendly lessons.</li>
</ul>



  </div>
</div>


      </div>
    </main>

    <footer>
  <div class="container">
    <div class="row">
      <div class="col-lg">
        <ul class="list-inline">
          
            <li class="list-inline-item">
              <a href="mailto:fpsom@certh.gr">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          

          
            <li class="list-inline-item">
              <a href="https://twitter.com/fopsom">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          

          
            <li class="list-inline-item">
              <a href="https://github.com/fpsom">
                <span class="fa-stack fa-lg">
                  <i class="fa fa-circle fa-stack-2x"></i>
                  <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                </span>
              </a>
            </li>
          
        </ul>
      </div>

      
      <div class="col-lg">
        


        
          <ul class="fa-ul">
          
            <li>
              <i class="fa-li fa fa-chevron-circle-right"></i>
              <a href="https://inab.certh.gr/" target="_blank">Institute of Applied Biosciences (INAB)</a>
            </li>
          
            <li>
              <i class="fa-li fa fa-chevron-circle-right"></i>
              <a href="https://www.certh.gr/" target="_blank">Centre for Research and Technology Hellas (CERTH)</a>
            </li>
          
          </ul>
        

      </div>
      
      <div class="col-lg">
        <p>This is the website for our research group. <br /> <a href="https://github.com/BiodataAnalysisGroup">Group GitHub</a> <br /> <a href="https://github.com/BiodataAnalysisGroup/BiodataAnalysisGroup.github.io/blob/jekyllCode/pages/code-of-conduct.md">Code of Conduct</a></p>


        

      </div>
      
    </div>

    <div class="small mt-2">
      
        <p>© Fotis Psomopoulos</p>
      
      
        <p>Content on this site is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0 License</a>. | Theme: <a href="https://github.com/peterdesmet/petridish">Petridish</a></p>

      
    </div>
  </div>
</footer>

<script src="/assets/theme/js/jquery.min.js"></script>
<script src="/assets/theme/js/popper.min.js"></script>
<script src="/assets/theme/js/bootstrap.min.js"></script>
<script src="/assets/theme/js/anchor.min.js"></script>

<script>
  // Show anchors when hovering over headings, using anchor.js
  (function () {
    const headings = ".content h2[id], .content h3[id], .content h4[id], .content h5[id], .content h6[id]";
    anchors.options = { placement: "right" };
    anchors.add(headings);
  })();
</script>



  </body>
</html>
