---
title: Our funded research projects
description: Listing our funded projects here
background: https://images.unsplash.com/photo-1572177812156-58036aae439c?auto=format&w=2000
permalink: /projects/
---
##  SYNTHIA

<img src="/assets/img/collaborations/synthia.jpg" alt="SYNTHIA" style="width:150px;"/>


[*Synthetic Data Generation framework for integrated validation of use cases and AI healthcare applications*](http://www.ihi-synthia.eu/)

- **Duration**: 2024-2029
- **Funder**: HORIZON-JU-IHI-2023-05-04
- **Description**: The SYNTHIA project, a groundbreaking public-private partnership set to revolutionize the field of personalized medicine by harnessing the power of synthetic data. As the first IHI project to tackle the critical need for privacypreserving data solutions in healthcare, SYNTHIA aims to propel research and innovation to new heights, ensuring that patients receive the best possible care while safeguarding their personal information. SYNTHIA is a multidisciplinary collaboration of 32 consortium partners - synthetic data generating developers, FAIR data experts, clinical researchers, developers of therapies and data-based tools, legal experts, socio-economic analysts, regulatory, policy advocacy, and communication experts - that will provide a 360º vision on how to advance healthcare applications through synthetic data use. Synthetic data, which are artificially generated to mimic real patient data, present a solution to many of the challenges faced in health research today. These data allow researchers to overcome the limitations of access to
high-quality, real-world datasets and address growing concerns about patient privacy. However, questions have lingered about the quality and applicability of synthetic datasets, especially in diverse and complex scenarios. SYNTHIA is addressing these challenges head-on by developing validated tools and methods for synthetic data generation across various data types, including laboratory results, clinical notes, genomics and imaging. By focusing on six specific diseases the project will demonstrate the utility of synthetic data in advancing personalized medicine: Lung cancer, Breast cancer, Multiple myeloma, Diffuse large B-cell lymphoma, Alzheimer’s disease and Type 2 diabetes.

<br>


##  EVERSE

<img src="/assets/img/collaborations/EOSC_Everse.jpg" alt="EVERSE" style="width:230px;"/>


[*European Virtual Institute for Research Software Excellence*](https://everse.software/)

- **Duration**: 2024-2027
- **Funder**: HORIZON-INFRA-2023-EOSC-01-02
- **Description**: The EVERSE project aims to create a framework for research software and code excellence, collaboratively designed and championed by the research communities across five EOSC Science Clusters and national Research Software Expertise Centres, in pursuit of building a European network of Research Software Quality and setting the foundations of a future Virtual Institute for Research Software Excellence. This framework for research software excellence will incorporate aspects involving community curation, quality assessment, and best practices for research software. This collective knowledge will be captured in the Research Software Quality toolkit (RSQkit), a knowledge base to gather and curate expertise that will contribute to high-quality software and code across different disciplines.

<br>

##  EOSC4Cancer

<img src="/assets/img/collaborations/EOSC4Cancer.png" alt="EOSC4Cancer" style="width:200px;"/>

[*A European-wide foundation to accelerate Data-driven Cancer Research*](https://eosc4cancer.eu/)

- **Duration**: 2023-2026
- **Funder**: HORIZON-INFRA-2021-EOSC-01-06
- **Description**: Cancer complex nature requires integration of advanced research data across national boundaries to enable progress. Indeed, the Horizon Europe mission board for cancer has identified access to data, knowledge and digital services - accessible across the European Research Area through federated infrastructures - as a key enabling condition for success. The better we organise cancer data across Europe, the better and faster we can bring the fruits of new biological and technical innovations to the benefit of EU citizens/patients. EOSC4Cancer will make cancer genomics, imaging, medical, clinical, environmental and socio-economics data accessible, using and enhancing existing federated and interoperable systems for securely identifying, sharing, processing and reusing FAIR cancer data across borders, and it will offer them via community-driven analysis environments. EOSC4Cancer provision of well curated datasets will be essential for advanced analytics and computational methods to be reproducible and robust, including machine learning and artificial intelligence approaches. EOSC4Cancer use-cases will cover the patient journey from cancer prevention to diagnosis to treatment, laying the foundation of data trajectories and workflows for future cancer mission projects. EOSC4Cancer brings together a comprehensive consortium of cancer research centres, research infrastructures, leading research groups, hospitals and supercomputing centres from 14 European countries. To make the developments sustainable, these will be offered as part of the research infrastructures partners services portfolio, in connection with the EOSC ecosystem and to serve the European Cancer Mission, which will be possible via the engagement with large international coalitions, e.g. ICGC-Argo, GA4GH, 1+MG/B1MG, Cancer Core Europe, European Cancer Information System, European Network of Cancer Registries, Innovative Partnership for Action Against Cancer Joint Action and patients/survivors associations.

<br>

##  SciLake

<img src="/assets/img/collaborations/scilake.png" alt="SciLake" style="width:120px;"/>

[*Democratising and making sense out of heterogeneous scholarly content*](https://scilake.eu/)

- **Duration**: 2023 – 2026
- **Funder**:  HORIZON-INFRA-2021-EOSC-01-04
- **Description**: SciLake’s mission is to build upon the OpenAIRE ecosystem and EOSC services to (a) facilitate and empower the creation, interlinking and maintenance of SKGs and the execution of data science and graph mining queries on top of them, (b) contribute to the democratization of scholarly content and the related added value services implementing a community-driven management approach, and (c) offer advanced, AI-assisted services that exploit customised perspectives of scientific merit to assist the navigation of the vast scientific knowledge space. In brief, SciLake will develop, support, and offer customisable services to the research community following a two-tier service architecture. First, it will offer a comprehensive, open, transparent, and customisable scientific data-lake-as-a-service (service tier 1), empowering and facilitating the creation, interlinking, and maintenance of SKGs both across and within different scientific disciplines. On top of that, it will build and offer a tier of customisable, AI-assisted services that facilitate the navigation of scholarly content following a scientific merit-driven approach (tier 2), focusing on two merit aspects which are crucial for the research community at large: impact and reproducibility. The services in both tiers will leverage advanced AI techniques (text and graph mining) that are going to exploit and extend existing technologies provided by SciLake’s technology partners. Finally, to showcase the value of the provided services and their capability to address current and anticipated needs of different research communities, four scientific domains (neuroscience, cancer research, transportation, and energy) have been selected to serve as pilots. For each, the developed services will be customised, to accommodate differences in research procedures, practices, impact measures and types of research objects, and will be validated and evaluated through real-world use cases.

<br>

## ML4NGP

<img src="/assets/img/collaborations/ML4NGP.png" alt="ML4NGP" style="width:150px;"/>

[*CA21160 - Non-globular proteins in the era of Machine Learning*](https://www.cost.eu/cost-action/non-globular-proteins-in-the-era-of-machine-learning/)

- **Duration**: 2022 - 2026
- **Funder**: [Cost - European Cooperation in Science & Technology](https://www.cost.eu/)
- **Description**: Protein structure prediction has long been considered the “Holy Grail” of structural biology. The recent success of AlphaFold has ushered in a new era of highly accurate structure prediction, bringing to light the secrets hidden in the three-dimensional structures of globular proteins, increasing our understanding about their structural features and molecular function. However, a large proportion of the proteomes from all domains of life are rich in sequences that do not fold into regular structures, commonly known as non-globular proteins (NGPs). NGPs comprise intrinsically disordered regions, repeats, low-complexity sequences, aggregation-prone and phase-separating sequences, and are implicated in a range of age-related diseases. Their heterogeneous structural states and low sequence complexity challenge current experimental structure determination techniques and machine learning (ML) methods for structure prediction, making the molecular understanding of their sequence-structure-dynamics-function relationship difficult. The recent improvements of ML approaches and advances in determining NGP structural ensembles call for a timely re-assessment of the interplay between experiments and computation. The ML4NGP Action aims to establish an interdisciplinary pan-European network to favour this interplay, fostering experimental frameworks designed to provide information to computational methods, and novel computational methods developed, trained and benchmarked with experimental data. ML4NGP will enhance the primary experimental data generation (WG1), promote integrative structural biology approaches (WG2), benchmark the state-of-the-art ML methods (WG3) and improve the functional characterization of NGPs (WG4). The Action will support its scientific objectives through policies that sustain free knowledge exchange, inclusiveness and training of young researchers who will lead future innovations in this field.

<br>

## GenOptics

<img src="/assets/img/collaborations/GenOptics_logo.png" alt="GenOptics" style="width:150px;"/>

[*Large Scale bio-data Visual analytics platform*](https://genoptics.github.io/)

- **Duration**: 2020 - 2023
- **Funder**: [General Secretariat for Research and Technology (GSRT)](http://www.gsrt.gr/)
- **Description**: GenOptics is a 30-month national project (2020-2023) that will implement a new platform for integrating, analyzing and visualizing multi-omics and other clinico-biological data, by extending the functionality of established bioinformatic analysis tools through an interactive visual dashboard. Visual analytics, namely the simultaneous use of computational data analysis tools with interactive visualizations, is a powerful way of combining algorithms with an intuitive, user-friendly interactive environment in order to extract knowledge from large scale bio-data. Although visual analytics could contribute to significantly reducing the gap between available data and discovery of new knowledge through targeted studies, there is no widely used platform that provides this functionality to the scientific community. GenOptics will allow visual analytics of multi-omics data through a modular system that will integrate multiple interoperable analysis tools into a software platform, leveraging established international technologies and software suites such as Docker and R/Bioconductor. 

<br>

## Gallantries

<img src="/assets/img/projects/gallantries-logo-with-word-inverted-large.png" alt="Gallantries" style="width:150px;"/>

[*When Galaxy meets Carpentries to develop and deliver open and scalable training in life sciences*](https://gallantries.github.io/)

- **Duration**: 2020 - 2023
- **Funder**: [Erasmus+](https://ec.europa.eu/programmes/erasmus-plus/node_en)
- **Description**: The [Gallantries project](https://gallantries.github.io/) has been funded by [Erasmus+](https://ec.europa.eu/programmes/erasmus-plus/node_en) to develop modular, broadcast-ready bioinformatics lessons over the next 3 years, aiming in particular for a hybrid form of training delivery. Partners and associated members of the project include several ELIXIR Nodes and Communities, and are dedicated to developing high quality bioinformatics training by combining the experience of Carpentries trainers with the Galaxy platform. Saskia Hiltemann & Helena Rasche ([ErasmusMC](https://erasmusmc.nl/), ELIXIR-NL), Fotis Psomopoulos ([INAB, CERTH](https://inab.certh.gr/), ELIXIR-GR), Bérénice Batut ([University of Freiburg](https://uni-freiburg.de/), ELIXIR-DE), Anthony Bretaudeau ([INRAE](https://www.inrae.fr/en), ELIXIR-FR), and Yvan Le  Bras ([MNHN](https://www.mnhn.fr/en), ELIXIR-FR) and their teams will be working together to develop, improve, and deliver quality, reusable, modular, and pandemic-friendly lessons.

<br>

## ELIXIR Commissioned Services

<img src="/assets/img/collaborations/ELIXIR.png" alt="ELIXIR" style="width:100px;"/>

Co-leading the following [ELIXIR](https://elixir-europe.org/) activities as commissioned services:
- Tools Platform Task 1 "Packaging, containerisation & deployment"
- Tools Platform Task 2 "Performance benchmarking & technical monitoring"
- Tools Platform Task 4 "_Software Best Practices_"
- Training Platform Task 2 "_Gap analysis, training materials development and training delivery_"
- Strategic Implementation Study "Making container services integratable, sustainable and widely used"
- Community-led Implementation Study "Improving IDP tools interoperability and integration into ELIXIR"

